app/services/solidus_pay_tomorrow/client/void_service.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+module SolidusPayTomorrow
+  module Client
+    class VoidService < SolidusPayTomorrow::Client::BaseService
+      attr_reader :order_token
+
+      CANCEL_ENDPOINT = 'api/ecommerce/application/:order_token/cancel'
```

Comment:
doesn't it need `.freeze` at the end?

Reply:
`frozen_string_literal: true` at the beginning of the file tells Ruby that all strings in the file are implicitly frozen.
Hence, freeze isn't needed for string types